src/plugins/intel_gna/src/transformations/remove_in_out_processing.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/remove_in_out_processing.hpp"
+
+#include "common/graph_utils.hpp"
+#include "openvino/cc/pass/itt.hpp"
+#include "openvino/opsets/opset1.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "openvino/opsets/opset7.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "openvino/pass/manager.hpp"
+#include "openvino/pass/pass.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/utils/transformation_helper.hpp"
+
+using namespace ov::opset10;
+using namespace ov::intel_gna::pass;
+
+namespace {
+
+inline bool is_preprocessing_layer_not_supported(std::shared_ptr<ov::Node>& layer) {
```

Comment:
is inline only to prevent missing declaration warning?
wouldn't be better to use `static`?